warehouse/ingest-scripts/src/main/resources/bin/ingest/load-job-cache.sh

```diff
@@ -42,9 +42,22 @@ sed s%${BEFORE}%${AFTER}% job-cache-env.sh > job-cache-env.tmp
 
 date
 
+function no_ctrlc() {
+  let count++
+  echo
+  if [[ $count == 1 ]]; then
+    echo "Are you sure you want to stop the load-job-cache.sh process? Giving you a warning."
```

Comment:
We should add some verbiage that describes what will occur because otherwise, you might not necessarily know the scope of the warning.

Maybe something like "Killing this script will leave the job cache in a most-likely unusable state. You will need to rerun this script."

Reply:
Oh, and then maybe clarify that you have to run `Ctrl+C` again to terminate.